.drone.yml

```
@@ -0,0 +1,13 @@
+name: default
+
+type: docker
+kind: pipeline
+
+steps:
+  - name: Build & Test
+    image: python:3
```

```suggestion
    image: python:3-alpine
```
If you only use python, I recommend using smaller images to speed up the process.

---

Okay, thank you!